aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java

```diff
@@ -52,13 +53,15 @@ static void predictTextSentimentAnalysis(String project, String content, String
     try (PredictionServiceClient predictionServiceClient =
         PredictionServiceClient.create(predictionServiceSettings)) {
       String location = "us-central1";
-      String jsonString = "{\"content\": \"" + content + "\"}";
+
+      JsonObject contentJsonObject = new JsonObject();
+      contentJsonObject.addProperty("content", content);
```

Comment:
Approving, but can you add a comment or link on why this safe serialization? Thanks!

Reply:
@averikitsch I sure can!

I'll be out with my mates watching a hockey game tonight; but I should be able to attend to this tomorrow.

Reply:
@wallind Your PR is approved and ready to merge, whenever you're ready with this final touch. Thanks!

Reply:
alright well it definitely wasn't _"tomorrow"_ heh; but here we are. I rebased with a comment included this time. Let me know what you think